ansible/modules/hashivault/hashivault_write.py

```diff
@@ -100,19 +103,23 @@ def hashivault_write(params):
         warnings.simplefilter("ignore")
         changed = True
         write_data = data
+
+        read_data = client.read(secret) or {}
```

Comment:
Double check my logic on this one, but I wanted to be able to determine changes even if I wasn't updating